MediaBrowser.XbmcMetadata/Savers/BaseNfoSaver.cs

```diff
@@ -230,6 +248,18 @@ private async Task SaveToFileAsync(Stream stream, string path)
             }
         }
 
+        private static async Task<byte[]> ComputeHash(Stream stream, CancellationToken cancellationToken)
+        {
+            byte[] hash;
+            using (SHA256 hasher = SHA256.Create())
```

Comment:
I opted for SHA256 here, but could easily be any other hasher. This method also could be extracted to some common library.

Reply:
Maybe we can use XxHash3 as in #11816 since it is way faster and still good enough

Reply:
Happy to swap to that. One moment.

Reply:
Now using XxHash3